3d_armor/api.lua

```diff
@@ -477,8 +478,9 @@ end
 
 armor.save_armor_inventory = function(self, player)
 	local _, inv = self:get_valid_player(player, "[save_armor_inventory]")
+	local meta = player:get_meta()	
 	if inv then
```

Comment:
Same as above. (There's no need to get `meta` if it's not used.)

Reply:
Is it fixed now?

Reply:
Is it fixed now?

Reply:
Yes, it's correct.

Reply:
Okay. If you find anything else, please tell me. Thanks! 👍